src/CitizenProject/CitizenProjectCommand.php

```diff
@@ -85,6 +85,20 @@ class CitizenProjectCommand
 
     public $skills;
 
+    /**
+     * @var UploadedFile|null
+     *
+     * @Assert\Image(
+     *     maxSize = "50M",
+     *     mimeTypes = {"image/jpeg", "image/png"},
+     *     minWidth = "1024",
+     *     minHeight = "576",
```

Comment:
by specifying `minWidth` and `minRatio`, you implictely declare a minimum height ?
is the statement `minHeight` useful here ?

Reply:
Not really :)